frontend/src/environments/environment.ts

```diff
@@ -4,8 +4,8 @@
 
 export const environment = {
```

Comment:
it is for production environment i guess so we should change it to production api url if we change it at all